ml-agents/mlagents/trainers/tests/simple_test_envs.py

```diff
@@ -322,3 +322,72 @@ def solve(self) -> None:
                 else:
                     self.action[name] = [[float(self.goal[name])]]
             self.step()
+
+
+class Simple2DEnvironment(Simple1DEnvironment):
+    def __init__(
+        self,
+        brain_names,
+        use_discrete,
+        step_size=0.5,
+        num_visual=0,
+        num_vector=1,
+        n_demos=30,
+    ):
+        self.positions: Dict[str, List[float]] = {}
+        super().__init__(
+            brain_names,
+            use_discrete,
+            step_size=step_size,
+            num_visual=num_visual,
+            num_vector=num_vector,
+        )
+        action_type = ActionType.DISCRETE if use_discrete else ActionType.CONTINUOUS
+        self.group_spec = AgentGroupSpec(
+            self._make_obs_spec(), action_type, (2, 2) if use_discrete else 2
+        )
+
+    def _take_action(self, name: str) -> bool:
```

Comment:
Since all these functions basically generalize that of the 1D env, could we just use this as the SimpleEnv and just pass a parameters for the dimensionality?